inference-engine/src/mkldnn_plugin/utils/README.md

```diff
@@ -0,0 +1,72 @@
+# Debug capabilities
```

Comment:
I was planning to add ENABLE_CPU_DEBUG_CAPS to https://github.com/openvinotoolkit/openvino/wiki/CMakeOptionsForCustomCompilation
And refer the .md file from there for more information.
But maybe it is better to have everything on the wiki, I am not sure.

Reply:
In general I would prefer the second approach. It is easier to maintain such description inplaced in the code. And I suppose it is easier to find by common developer. 

@ilya-lavrenov  Do you have any concerns?

Reply:
The reason I think it is better to have the documentation under version control is that it will be connected with the actual implementation, so developers could match documentation with the current commit in case we update / change functionality.

Reply:
I'm Ok, my comment was just a recommendation.

Reply:
@dmitry-gorokhov So I remove the .md file and update wiki after we merge this, right?

Reply:
@EgorDuplensky No. Lets keep md and update the wiki with reference on md.